arcaflow_plugin_elasticsearch/es_plugin.py

```diff
@@ -7,7 +7,11 @@
 from elasticsearch import Elasticsearch
 
 from arcaflow_plugin_sdk import plugin
-from es_schema import ErrorOutput, SuccessOutput, StoreDocumentRequest
+from .es_schema import (
```

Comment:
What changed to require the lead `.`?

Reply:
I receive error `ModuleNotFoundError: No module named 'es_schema'` without this method while testing

Reply:
Try adding an empty `__init__.py` along side those files.  It makes that directory a proper python module.